sdks/java/fn-execution/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   shadow library.java.grpc_stub
   shadow library.java.grpc_netty
   shadow library.java.netty_transport_native_epoll
+  shadow library.java.slf4j_api
```

Comment:
This is already a dependency, see line 29

Reply:
rm'd

Added an implementation in the testing scope